extensions/ql-vscode/src/remote-queries/shared/analysis-result.ts

```diff
@@ -7,12 +7,12 @@ export interface AnalysisResults {
 }
 
 export interface AnalysisAlert {
-  message: string;
+  message: AnalysisMessage;
   shortDescription: string;
   severity: ResultSeverity;
   filePath: string;
   codeSnippet: CodeSnippet;
-  highlightedRegion: HighlightedRegion;
+  highlightedRegion?: HighlightedRegion;
```

Comment:
I've made this optional because it made my code simpler but also because I think it makes sense for this to be optional. We won't always want to highlight a specific region.